scan_filter_chain:
- name: shadows
  type: ScanShadowsFilter
  params:
    min_angle: 10
    max_angle: 170
    neighbors: 18
    window: 1
